plugins/taoqtiimage/lang/nl.js

```diff
@@ -0,0 +1,4 @@
+CKEDITOR.plugins.setLang( 'insertQtiImage', 'nl', {
+	button: 'bijsluiterafbeelding',
+	title: 'bijsluiterafbeelding'
```

Comment:
Where did you get this translation from?
I'm pretty sure those should be 2 words.
```suggestion
	button: 'bijsluiter afbeelding',
	title: 'bijsluiter afbeelding'
```

Reply:
@KRAMAOAT tell me. I will request again for confirm. Thanks

Reply:
Confirmated